static/js/components/tiles/place_overview_tile.tsx

```diff
@@ -32,11 +34,48 @@ interface PlaceOverviewTilePropType {
 export function PlaceOverviewTile(
   props: PlaceOverviewTilePropType
 ): JSX.Element {
+  const [subtopics, setSubtopics] = useState(null);
+
+  useEffect(() => {
+    axios
+      .get(`/api/landingpage/data/${props.place.dcid}?category=Overview&hl=en`)
+      .then((resp) => {
+        const categories = Object.keys(resp.data.categories);
+        const subtopics = categories.filter(
+          (category) => category !== "Overview"
+        );
+        setSubtopics(subtopics);
+      });
+  }, []);
+
+  if (subtopics === null) {
+    return null;
+  }
+
   return (
-    <div className="chart-container place-overview-tile">
-      <RawIntlProvider value={intl}>
-        <Overview dcid={props.place.dcid} locale="en" />
-      </RawIntlProvider>
-    </div>
+    <>
+      <div className="chart-container place-overview-tile">
+        <RawIntlProvider value={intl}>
+          <Overview dcid={props.place.dcid} locale="en" />
+        </RawIntlProvider>
+      </div>
+      {!_.isEmpty(subtopics) && (
+        <div className="subtopics-section">
+          <h3>See more about:</h3>
```

Comment:
maybe: "Learn more about <place name>:"
then the categories as a comma separated list of links (so it's sentenc-y)

Reply:
so the links all on one line?